baseline/OSPS-BR.yaml

```
@@ -387,6 +404,10 @@ controls:
           - reference-id: RA-5
           - reference-id: SA-15
           - reference-id: SR-3
+      - reference-id: UKSSCOP
+        entries:
+          - reference-id: Claim 2.1.2
+          - reference-id: Claim 3.1.1  
```

```suggestion
          - reference-id: Claim 1.2.2.1.1
```

I think the intent here more closely matches "Integrity of third-party components and updates is validated"

---

Your suggestion didn't come through clearly "1.2.2.1.1" maybe means "1.2.1"?

---

1.2.5 seems like it should apply as well.